packages/design-system/src/SegmentedControl/SegmentedControl.types.ts

```diff
@@ -22,4 +22,6 @@ export type SegmentedControlProps = {
   defaultValue?: string;
   /** The onChange of the SegmentedControl */
   onChange?: (value: string) => void;
+  /** Whether the control should consume full width of container or not */
```

Comment:
Please add a TODO talking about why this is a bad idea but why it has been done anyway and what is needed to refactor this.